src/appleseed.cli/stdouttilecallback.cpp

```diff
@@ -68,9 +68,9 @@ namespace
       : public TileCallbackBase
     {
       public:
-        StdOutTileCallback(const bool single_plane = false)
-          : m_single_plane(single_plane)
-          , m_header_sent(false)
+        explicit StdOutTileCallback(TileOutputOptions export_options)
+          : m_header_sent(false)
+          , m_single_plane(export_options == BeautyOnly)
```

Comment:
I'd go as far as storing export options directly instead of having `m_single_plane` that doesn't much to me.